### DISCUSSION TITLE
Consulta ci/cd

### DISCUSSION THREAD

**Discussion by NicoPavoni** [frlp-utn-ingsoft/catedra#149] [Q&A] [answered]
Buenas! Tengo una consulta, en una branch aparte agregamos ci/cd, recien hicimos el PR a main y cuando corrió los checks, uno fallo

![2024-06-04 12_52_16-Se agrega workflow ci_cd · NicoPavoni_vetsoft@65655d7 - Opera](https://github.com/frlp-utn-ingsoft/catedra/assets/104012558/4abc9234-e979-4cdc-b79e-bf6d6c1a9194)

Básicamente hay unos imports y variables declaradas que no se usan. En este caso, como hacemos ? corregimos esos puntos, commiteamos, pusheamos y volvemos a correr el PR ? Y estas modificaciones, las hacemos sobre la misma branch de ci/cd o en una aparte?

Gracias

**Comment by RodrigoJacznik** [comment, marked as answer]
Buenas, el ci/cd mergealo con main. Asi ya queda.
Los fix de ruff van sobre la rama que activa esa regla de ruff.
